### PR TITLE
Manpage

### PR DESCRIPTION
**Pull request by somers-all-the-time**
autorandr.1 is the initial manpage written in groff markup language

### CODE REVIEW

**Comment by phillipberndt** [comment]
Thanks, much appreciated! Should we also add this to the Makefile as a `manpage` target that is included by default?

**Comment by somers-all-the-time** [comment]
Does this work?

File: Makefile [hunk 1]
```diff
@@ -121,6 +121,18 @@ uninstall_udev:
 	$(if $(UDEV_RULES_DIR),,$(error UDEV_RULES_DIR is not defined))
 	rm -f ${DESTDIR}/${UDEV_RULES_DIR}/40-monitor-hotplug.rules
 
+# Rules for manpage
+MANPAGES_DIR:=/usr/share/man/man1
```
Comment (phillipberndt) [inline Makefile]: This variable is typically called `MANDIR`, and it should be defined relative to `PREFIX`, that is `${PREFIX}/share/man/man1`.

File: Makefile [hunk 2]
```diff
@@ -121,6 +121,18 @@ uninstall_udev:
...
+DEFAULT_TARGETS+=manpage
+
+install_manpage:
+	mkdir -p ${MANPAGES_DIR}
```
Comment (phillipberndt) [inline Makefile]: Here and below the variable should be prefixed with `${DESTDIR}/`.

**Comment by phillipberndt** [comment]
Merged, thanks!

**Pull request merged**